src/payment/strategies/paypal-commerce/paypal-commerce-overlay.ts

```diff
@@ -0,0 +1,42 @@
+import { Overlay, OverlayShowOptions } from '../../../common/overlay';
+
+export const idOverlay = 'paypal-commerce-overlay';
+
+export default class OverlayPaypal extends Overlay {
+
+    constructor() {
+        super({ id: idOverlay });
+    }
+
+    show(options?: OverlayShowOptions) {
+        super.show(options);
+        const element = document.getElementById(idOverlay);
+
+        if (element) {
+            const modal = document.createElement('div');
```

Comment:
pretty minor, but in order to avoid the indentation of the whole block, you could do instead

```js
if (!element) {
  return;
}

const text = document.createElement('div');
const link = document.createElement('a');
// ...
```